backend/src/permissions/permissions.service.spec.ts

```diff
@@ -727,35 +751,19 @@ describe('PermissionsService', () => {
     ) as Group;
     const note = Note.create(user) as Note;
     it('emits PERMISSION_CHANGE event', async () => {
-      jest
-        .spyOn(noteRepo, 'save')
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        .mockImplementationOnce(async (entry: Note) => {
-          return entry;
-        });
-      const mockedEventEmitter = jest
-        .spyOn(eventEmitter, 'emit')
-        .mockImplementationOnce((event) => {
-          expect(event).toEqual(NoteEvent.PERMISSION_CHANGE);
-          return true;
-        });
-      expect(mockedEventEmitter).not.toHaveBeenCalled();
+      expect(emitEvent).not.toHaveBeenCalled();
       await service.updateNotePermissions(note, {
         sharedToUsers: [],
         sharedToGroups: [],
       });
-      expect(mockedEventEmitter).toHaveBeenCalled();
+      expect(emitEvent).toHaveBeenCalled();
     });
     describe('works', () => {
+      beforeEach(() => {
+        mockNoteRepo(noteRepo);
+        emitEvent;
+      });
```

Comment:
This is already handled by the previous beforeEach you've inserted.

Reply:
removed